### PR TITLE
docs(self-managed): zeebe gateway auth is enabled by default in 8.2

### PR DESCRIPTION
**Pull request by aabouzaid**
## Description

Zeebe Gateway auth has been enabled by default in 8.2.

Related to: https://github.com/camunda/camunda-platform-helm/issues/695

## When should this change go live?

Anytime (ASAP).

- [ ] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [x] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [x] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [x] I have added changes to the relevant `/versioned_docs` directory.
- [x] I have added changes to the main `/docs` directory (aka `/next/`).
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer.

### CODE REVIEW

File: versioned_docs/version-8.2/self-managed/zeebe-deployment/security/client-authorization.md [hunk 1]
```diff
@@ -7,11 +7,11 @@ description: "Learn how the Zeebe Gateway supports Camunda Identity-based auth t
 ## Gateway
 
 The Zeebe Gateway supports [Camunda Identity](../../identity/what-is-identity.md)-based auth token validation.
-By default, authentication is disabled by setting `security.authentication.mode` to `none`.
+In Camunda Platform 8 Self-Managed Helm chart, the authentication is enabled via Camunda Identity like all other applications.
 
 ### Camunda Identity authorization
 
-[Camunda Identity](../../identity/what-is-identity.md)-based OAuth token validation can be enabled by setting `security.authentication.mode` to `identity` and providing the corresponding `security.authentication.identity.*` properties. You can find more details about these in the [Gateway config documentation](../configuration/gateway.md#zeebegatewayclustersecurityauthenticationidentity).
+[Camunda Identity](../../identity/what-is-identity.md)-based OAuth token validation is already enabled by setting `security.authentication.mode` to `identity` and providing the corresponding `security.authentication.identity.*` properties. You can find more details about these in the [Gateway config documentation](../configuration/gateway.md#zeebegatewayclustersecurityauthenticationidentity).
```
Comment (megglos) [inline versioned_docs/version-8.2/self-managed/zeebe-deployment/security/client-authorization.md:14]: 💭 Would leave this paragraph as is, just explaining how it's configured but add another one that hints about the default in the helm chart, wdyt?
```suggestion
[Camunda Identity](../../identity/what-is-identity.md)-based OAuth token validation can be enabled by setting `security.authentication.mode` to `identity` and providing the corresponding `security.authentication.identity.*` properties. You can find more details about these in the [Gateway config documentation](../configuration/gateway.md#zeebegatewayclustersecurityauthenticationidentity).

In the Camunda Platform 8 Self-Managed Helm chart it's already fully preconfigured by default.
```

**Review by megglos — APPROVED** [review]
LGTM! Thanks for improving this page @aabouzaid ! Just minor suggestions, let's best await @christinaausley's feedback first.

File: docs/self-managed/zeebe-deployment/security/client-authorization.md [hunk 2]
```diff
@@ -5,11 +5,11 @@ description: "Learn how the Zeebe Gateway supports Camunda Identity-based auth t
 ---
 
 The Zeebe Gateway supports [Camunda Identity](../../identity/what-is-identity.md)-based auth token validation.
-By default, authentication is disabled by setting `security.authentication.mode` to `none`.
+In Camunda Platform 8 Self-Managed Helm chart, the authentication is enabled via Camunda Identity like all other applications.
 
 ### Camunda Identity authorization
 
-[Camunda Identity](../../identity/what-is-identity.md)-based OAuth token validation can be enabled by setting `security.authentication.mode` to `identity` and providing the corresponding `security.authentication.identity.*` properties. You can find more details about these in the [Gateway config documentation](../configuration/gateway.md#zeebegatewayclustersecurityauthenticationidentity).
+[Camunda Identity](../../identity/what-is-identity.md)-based OAuth token validation is already enabled by setting `security.authentication.mode` to `identity` and providing the corresponding `security.authentication.identity.*` properties. You can find more details about these in the [Gateway config documentation](../configuration/gateway.md#zeebegatewayclustersecurityauthenticationidentity).
```
[the remark this answers is not in the document]
Comment (aabouzaid) [reply docs/self-managed/zeebe-deployment/security/client-authorization.md]: I think no need to mention the Helm chart config away from [its section](https://docs.camunda.io/docs/next/self-managed/platform-deployment/helm-kubernetes/overview/).
This page is about Zeebe and the possible options in it.

**Comment by aabouzaid** [comment]
@christinaausley FYI, this PR is ready for review.

File: docs/self-managed/zeebe-deployment/security/client-authorization.md [hunk 3]
```diff
@@ -5,11 +5,11 @@ description: "Learn how the Zeebe Gateway supports Camunda Identity-based auth t
 ---
 
 The Zeebe Gateway supports [Camunda Identity](../../identity/what-is-identity.md)-based auth token validation.
-By default, authentication is disabled by setting `security.authentication.mode` to `none`.
+In Camunda Platform 8 Self-Managed Helm chart, the authentication is enabled via Camunda Identity like all other applications.
```
[the remark this answers is not in the document]
Comment (korthout) [reply docs/self-managed/zeebe-deployment/security/client-authorization.md]: @megglos (or @aabouzaid) Can you answer this? I understand that the Helm charts are the recommended way to install a production cluster, but our bare-metal configuration still has a different default. I feel that we should highlight that this difference exists.

**Comment by akeller** [comment]
@aabouzaid are you looking to get this into tomorrow's alpha release? Christina is on FTO but I'm reviewing PRs and want to know how I should prioritize this one.

**Comment by aabouzaid** [comment]
@akeller yes this PR is a bit critical. It can wait, but it increases the chances to get more support tickets about this topic.
So it would be great to have it in the alpha release especially it's tiny change.

**Review by akeller — APPROVED** [review]
Small fixes in review commit, nothing major.

**Comment by akeller** [comment]
@aabouzaid there are a few unresolved comments you may want to take a look at, but you have the 👍 to merge from me if you'd like to address those in a future PR.

**Pull request merged**